tests/test_utils.py

```
@@ -2,325 +2,304 @@
         assert success is True
-        assert output == 'Ready'
-        assert error == ''
-    
-    @patch('rhoshift.utils.utils.run_command')
-    @patch('time.sleep')
+        assert output == "Ready"
+        assert error == ""
+
+    @patch("rhoshift.utils.utils.run_command")
+    @patch("time.sleep")
     def test_wait_for_resource_timeout(self, mock_sleep, mock_run_command):
```

Ruff (0.13.1)

Unused method argument: `mock_sleep`

(ARG002)